devops/jobs/CreateSandbox.groovy

```diff
@@ -203,6 +203,9 @@ class CreateSandbox {
                              "video_pipeline and video_encode_worker must be selected for video pipeline to work")
                 stringParam("video_encode_worker_version","master","")
 
+                booleanParam("prospectus",false,"Enable Prospectus, the edx cataolog")
+                stringParam("prospectus_version","master","")
```

Comment:
out of curiosity what is that third argument?

Reply:
a description of what the param is for. Other services didn't have one so 🤷